ovirt_imageio/client/_options.py

```diff
@@ -178,6 +178,14 @@ class Parser:
             default="warning",
             help=(f"Log level (choices: {log_level}, default: warning)."),
         ),
+        Option(
+            name="description",
+            args=["-d", "--description"],
+            config=True,
+            required=False,
+            type=str,
+            default="Uploaded by ovirt-img",
```

Comment:
Since this is repeated a couple times, it could be a global variable similar to other default values in other options.